news/565.api_change

```diff
@@ -0,0 +1,2 @@
+`OmegaConf.select()` as well as container methods `get()` and `pop()` do not return their default value anymore when the accessed key is an interpolation that cannot be resolved: instead, an exception is raised.
```

Comment:
```suggestion
`OmegaConf.select()` , `DictConfig.{get(), pop()}`, `ListConfig.{get(),pop()}` no longer returns the specified default value when the accessed key is an interpolation that cannot be resolved: instead, an exception is raised.
```

Reply:
Done in a851a91 (almost your suggestion, up to a few minor formatting fixes)